modules/embit/embit_lib/embit_lib.cpp

```diff
@@ -43,39 +49,76 @@ static bool call_python_parser(const char* input, const char* function_name) {
         goto cleanup;
     }
     
-    // Add the input string to the tuple
-    if (PyTuple_SetItem(args, 0, input_str) < 0) {
+    // Add the input object to the tuple
+    if (PyTuple_SetItem(args, 0, input_obj) < 0) {
         goto cleanup;
     }
-    input_str = NULL;  // Ownership transferred to args
+    input_obj = NULL;  // Ownership transferred to args
     
     // Call the Python function
     result = PyObject_CallObject(parse_func, args);
     if (!result) {
+        if (function_name && strcmp(function_name, "psbt_parse") == 0) {
+            PyErr_Print(); // only printing err for psbt_parse
+        }
         goto cleanup;
     }
```

Comment:
We can remove this print.
```suggestion
    result = PyObject_CallObject(parse_func, args);
    if (!result) {
        goto cleanup;
    }
```